chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/SummaryStatisticsCreatorJob.java

```diff
@@ -152,8 +152,6 @@ public void saveSummaryStatistics(EmailStatistics statistics)
             @Override
             protected void doInTransactionWithoutResult(TransactionStatus status) {
                 try {
-                    summaryStatisticsDAO.deleteAll();
```

Comment:
I noticed the data model PR un-deleted all the stats data gathered for this year so we will have a few months of stats data to start with I guess? What is the longer term plan? Do we never mark stats data as deleted or do we want this job to keep a rolling <time window> of data un-deleted but mark anything else as deleted?

Reply:
I was under the impression we would no longer be marking Summary Stats as deleted.

Reply:
I think the idea was that we would no longer mark things as deleted going forward, as we need some historical data, but we didn't want to undelete _everything_, because we're not sure how the old format of data would work if it were used again (somehow), but stuff from the first of the year is probably safe to use